### PR TITLE
En 1300 ux patterns

### PR DESCRIPTION
**Pull request by nicpuddu**

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

File: src/ui/dashboard/DashboardPage.js [hunk 1]
```diff
@@ -10,6 +11,9 @@ const DashboardPage = () => (
         <Col md={4}>
```
Comment (fedelocci) [inline src/ui/dashboard/DashboardPage.js]: use xs attribute

File: src/ui/dashboard/DashboardPage.js [hunk 2]
```diff
@@ -10,6 +11,9 @@ const DashboardPage = () => (
...
           <UserManagementContainer />
         </Col>
+        <Col md={4}>
```
Comment (fedelocci) [inline src/ui/dashboard/DashboardPage.js]: use xs attribute

Comment (nicpuddu) [reply src/ui/dashboard/DashboardPage.js]: why? ux is fine with those collapsing one on top of each other

**Pull request merged**